routes/net-state-files.go

```diff
@@ -0,0 +1,79 @@
+package routes
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/storage/boltdb"
+)
+
+var (
+	ErrReadReq = errors.New("error reading request body")
+)
+
+type File struct {
```

Comment:
lots of things called file. Maybe this is `type NetStateRoutes struct`? or if it's in a subpackage `netstate`, just `type Routes struct`?